test/export/test_export.py

```diff
@@ -11881,6 +11881,30 @@ def forward(self, x):
             ]
             self.assertEqual(len(shift_op), 1)
 
+    def test_wrapper_module(self):
+        def f(x):
+            return torch.abs(x)
+
+        from torch.export import _trace
+
+        model = _trace._WrapperModule(f)
```

Comment:
I would prefer to move this to some utils file, instead of skipfiling the entire trace.py